nbs/index.ipynb

```diff
@@ -79,9 +79,7 @@
     "It is strongly recommended to install AlphaPept in its own environment.\n",
     "1. Open the console and create a new conda environment: `conda create --name alphapept python=3.8`\n",
     "2. Activate the environment: `conda activate alphapept`\n",
-    "3. Redirect to the folder of choice and clone the repository: `git clone https://github.com/MannLabs/alphapept.git`\n",
-    "4. Navigate to the alphapept folder with `cd alphapept` and install the package with `pip install .` (default users) or with `pip install -e .` to enable developers mode.\n",
-    "5. To ensure compatibility, install strict requirements with `pip install -r requirements.txt`.\n",
+    "3. Install AlphaPept via pip: `pip install \"alphapept[gui-stable]\"`. If you want to use AlphaPept as a package withouth the GUI dependencies and strict version dependencies use `pip install alphapept`.\n",
```

Comment:
Dedinately mention all options and their intended use cases (e.g. AlphaTims states: 
![image](https://user-images.githubusercontent.com/11159117/142992225-f5590b49-b98b-42bd-9b95-aef75fe29c99.png)
)